package.json

```diff
@@ -104,7 +104,7 @@
     "ember-bunsen-core": "0.24.0",
     "ember-cli-babel": "^5.1.8",
     "ember-cli-htmlbars": "^1.0.3",
-    "ember-frost-core": "1.12.0",
+    "ember-frost-core": "1.13.0",
```

Comment:
Changing the min version of a dependency should not happen in a patch. Since it isn't a major version bump of `ember-frost-core` I'd say it is safe to make this PR a minor.